physics/dcyc2t3.f

```diff
@@ -247,10 +247,12 @@ subroutine dcyc2t3_run                                            &
 
 !  ---  outputs:
       real(kind=kind_phys), dimension(:), intent(out) ::                &
-     &      adjsfcdsw, adjsfcnsw, adjsfcdlw, adjsfculw, xmu, xcosz,     &
+     &      adjsfcdsw, adjsfcnsw, adjsfcdlw, xmu, xcosz,                &
      &      adjnirbmu, adjnirdfu, adjvisbmu, adjvisdfu,                 &
      &      adjnirbmd, adjnirdfd, adjvisbmd, adjvisdfd
 
+      real(kind=kind_phys), dimension(:), intent(inout) :: adjsfculw
```

Comment:
From what I can tell, this variable is not explicitly set here. Only printed in an if statement. So since it's coming in from elsewhere it would need 'INOUT' to help it pass through here.